BUILD.md

```diff
@@ -318,20 +326,33 @@ DOCKER_TRACE=1 bin/docker-build-proxy
 ### Updating protobuf dependencies
  If you make Protobuf changes, run:
  ```bash
-bin/dep ensure
```

Comment:
Do you also want to remove the `bin/dep ensure` instructions on line 265 in the Web section?